src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
+@import 'utils/variables';
+@import 'utils/mixins';
+@import 'blocks/header';
+@import 'blocks/nav';
+@import 'blocks/card';
+@import 'blocks/stars';
+
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  color: $main-text-color;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+a {
+  cursor: pointer;
 }
+
```

Comment:
Consider moving reset and general styles to a separate file to keep your main file clean and clear